Sources/Search/APIs/Custom.php

```diff
@@ -27,6 +32,24 @@
  */
 class Custom extends SearchApi implements SearchApiInterface
 {
+	/**************************
+	 * Public static properties
+	 **************************/
+
+	/**
+	 * @var array
+	 *
+	 * Sub-actions to add for SMF\Actions\Admin\Search::$subactions.
+	 */
+	public static array $admin_subactions = [
+		'createmsgindex' => __CLASS__ . '::build',
+		'removecustom' => __CLASS__ . '::remove',
+	];
```

Comment:
Can this be a method?

I have these three methods in my  custom action interface. Perhaps they could be of use.

	public function addSubAction(string $sa, callable $cb): void;
	public function hasSubAction(string $sa): bool;
	public function callSubAction(string $sa): void;

Reply:
🤔 Maybe. Those methods do look interesting. 

Since you already have code and an idea for this, how about I just leave it to you to make a PR implementing that idea at your leisure after this is merged?